src/components/CourseInfo/CourseInfo.jsx

```diff
@@ -4,8 +4,11 @@ import { LOCALES } from '../../constants';
 import { formatCreationDate, getCourseDuration } from '../../helpers';
 import { Link, useParams } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
 
-const CourseInfo = ({ courses }) => {
+const CourseInfo = () => {
+	//for some reason 'courses' is empty here. But this same array is not empty in 'Courses.jsx'. Why?
+	const courses = useSelector((state) => state.courses);
```

Comment:
`courses` are empty because currently you need to reopen web app by manually inputing url in order to see course info page. When you do so, you need to wait until courses are loaded and extract the required course after that.